docker/spark-mesos/Dockerfile

```diff
@@ -17,14 +17,18 @@
 # limitations under the License.
 #
 
-FROM mesosphere/mesos:0.20.1
+# Note: This is just a template. You need to replace <SPARK_VERSION> and <HADOOP_VERSION> 
```

Comment:
@tnachen are there other places in the docs where we should document this? Is there a section for "running Mesos with docker" somewhere?